ansible/roles/commcarehq/templates/localsettings.py.j2

```diff
@@ -164,7 +164,7 @@ ANALYTICS_IDS = {
 
 AXES_LOCK_OUT_AT_FAILURE = False
 
-LUCENE_ENABLED = True
+LUCENE_ENABLED = False
```

Comment:
@TylerSheffels why did you need to change this? This breaks the search functionality on couchlog.